src/app/external-cluster-wizard/steps/external-cluster/provider/aks/component.ts

```diff
@@ -73,21 +90,25 @@ export class AKSClusterSettingsComponent
   implements OnInit, OnDestroy, ControlValueAccessor, Validator
 {
   readonly Controls = Controls;
+  readonly Mode = Mode;
   readonly ErrorType = ErrorType;
   readonly AUTOSCALING_MIN_VALUE = 1;
   readonly AUTOSCALING_MAX_VALUE = 1000;
-
+  @Input() projectID: string;
+  @Input() cluster: ExternalCluster;
   isLoadingVmSizes: boolean;
+  isLoadingNoodPoolVersions: boolean;
   vmSizes: string[] = [];
-
-  @Input() projectID: string;
+  kubernetesVersions: string[] = [];
```

Comment:
can we rename to since method name indicates we are fetching AKS node pool versions
```suggestion
 nodePoolVersions string[] = [];
```